### ISSUE TITLE
Uploading CI data to GCS can be slow due to missing `crcmod` issues

### ISSUE THREAD

**Issue by phlax** [envoyproxy/envoy#22101]
Currently uploading docs can take a while (i guess similar with coverage and other upload jobs)

seems like this warning gives clues why:

```console


WARNING: gsutil rsync uses hashes when modification time is not available at
both the source and destination. Your crcmod installation isn't using the
module's C extension, so checksumming will run very slowly. If this is your
first rsync since updating gsutil, this rsync can take significantly longer than
usual. For help installing the extension, please see "gsutil help crcmod".

```

i looked at this a long time ago and i think i got the crcmod part resolved but dont rem exactly

**Comment by ankatare** [comment]
@phlax Clue to repro it would be helpful.

**Comment by phlax** [comment]
check CI - eg here https://dev.azure.com/cncf/envoy/_build/results?buildId=113266&view=logs&j=91e633a5-4907-5da1-5862-92a79fe2387a&t=4e621882-ec00-57a9-5411-d6f4a88c2fed

**Comment by ankatare** [comment]
@phlax Ok

**Comment by phlax** [comment]
fixed in #26399
